spacy_llm/registry/reader.py

```diff
@@ -6,6 +6,23 @@
 from .util import registry
 
 
+@registry.misc("spacy.JinjaReader.v1")
```

Comment:
I feel like this has moved away from how it was originally envisioned and should  probably be turned into a more generic file reader instead? All it does is require a particular file ending and then read in a string.

And in terms of the file extensions it seems overly restrictive for jinja where there are multiple file endings in common use? (spacy uses `.jinja`)